front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/ItineraryMarkers.tsx

```diff
@@ -57,9 +58,13 @@ const formatPointWithNoName = (
   </>
 );
 
-const extractMarkerInformation = (pathSteps: (PathStep | null)[], showStdcmAssets: boolean) =>
-  pathSteps.reduce((acc, cur, index) => {
-    if (cur && cur.coordinates) {
+const extractMarkerInformation = (
+  pathSteps: (PathStep | null)[],
+  showStdcmAssets: boolean,
+  invalidItems: string[] = []
+) => {
+  return pathSteps.reduce((acc, cur, index) => {
+    if (cur && cur.coordinates && (!('trigram' in cur) || !invalidItems.includes(cur.trigram))) {
```

Comment:
This condition is enough: if you don't find the pathItem in the infra, you won't have any coordinates. You can remove invalidItems from this component
```suggestion
    if (cur && cur.coordinates) {
```

Reply:
done